ChangeLog.md

```diff
@@ -8,13 +8,23 @@ Major changes:
 
 Behaviour changes:
 
+* `stack setup` no longer uses different GHC configure options on Linux
```

Comment:
I don't see this in the change.  Is it part of making the tarballs?  I'd expect it to be conditional on ghc >= 8, so that'd be consistent with it being part of tarball creation.

Reply:
It's the removal of the code for the `nopie` flag: https://github.com/commercialhaskell/stack/pull/3715/files#diff-4968f87e674615592e572f2ec593e5b0L584

That was put in place when nopie/tinfo6/ncurses6 changes were very in flux in the distros 
 (especially Arch) and that approach ended up not being very future proof.  The new approach is to patch `configure` in the GHC bindists to detect this condition and set the GHC `settings` file correctly.